pages/docs/tutorials/create-asyncapi-document.md

```diff
@@ -0,0 +1,167 @@
+
+Let's pretend you have a company called Smarty Lighting, and you install smart-city streetlight lighting systems. This smart lighting system is a use case of the Internet of things (IoT).You will create a Smartylighting Streetlights API using Node.js and Mosquitto (MQTT) as the message broker. This API will allow you to manage city lights remotely. 
+
+You want to build a system that can turn streetlights on and off based on their environmental conditions: 
+
+- You will implement an event-driven architecture (EDA) with a message broker in its "center."
+
+- Streetlights will publish information about their environmental lighting to the broker.
+
+- Your application will connect to the broker and receive a stream of events from all the streetlights reporting their conditions.
+
+- Your application decides based on events when to turn the streetlight off.
+
+- Your application is not aware of how many streetlights are publishing events - it just connects to the broker and receives all events.
+
+
+## Background context
+
+Event-driven architecture (EDA) is a design pattern built around the production, detection, and reaction to events that take place in time. In this pattern, a message broker, event publishers and subscribers are its main components for event exchange within microservices.
+
+[Message brokers](/docs/tutorials/getting-started/event-driven-architectures#message-broker) enables asynchronous communications between services so that the sending service need not wait for the receiving service’s reply. This allows interdependent services to “talk” with one another directly, even if they were written in different languages or implemented on different platforms. 
+
+Futhermore, the [Pub/sub](/docs/tutorials/getting-started/event-driven-architectures#publishersubscriber) is appealing for IoT use cases due to two key features: support for flexible coupling between publishers/subscribers and inherent support for point-to-multipoint transmission.  
+
+[MQTT](https://mqtt.org/), is a well-known protocol that is widely used in IoT applications because it was created particularly to address machine-to-machine (M2M) communication.
+
+## Create AsyncAPI document
+
+In this step, you will create an AsyncAPI document to describe the Streelights API. It will help you generate the code and the documentation later on.
+
+To create one, you can either use the [AsyncAPI Studio](https://studio.asyncapi.com) or the [AsyncAPI CLI](https://github.com/asyncapi/cli), depending on your project need.
+
+<Remember>
+
+If you do not have an `asyncapi.yaml` file ready, then generate one by running: 
+`asyncapi new --example=tutorial.yml --no-tty`.
+
+Next, generate a server by running:
+
+    asyncapi generate fromTemplate asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto
+    cd output && npm install
```

Comment:
```suggestion
You can also generate `asyncapi.yaml` document by running: 
`asyncapi new --example=tutorial.yml --no-tty`.
```

it could not be copied 1:1
this tutorial is about creating asyncapi document, no generation should be mentioned

@alequetzalli this is only thing that needs to be adjusted and we can accept

Reply:
You wrote "this tutorial is about creating asyncapi document, **no generation should be mentioned**", but then you proceed to add a commit with the word `generate` 😜  ... nice going there @derberg ... 😂

<img width="995" alt="Screen Shot 2023-02-08 at 6 43 40 PM" src="https://user-images.githubusercontent.com/19964402/217704343-002d7abe-676d-4058-ab65-500032ab1d80.png">